### ISSUE TITLE
Add pilots or owners stats

### ISSUE THREAD

**Issue by Ysurac** [Ysurac/FlightAirMap#109]
Pilot for IVAO/VATSIM, owner for real aircrafts.
Stats that can be done :
* Most common pilots or owners
* ?

**Comment by tonkepen100** [comment]
Really do not understand the question, please explain it more clearly and help you

**Comment by Ysurac** [comment]
I don't know what others stats I can put for pilots and/or owners.

**Comment by tonkepen100** [comment]
For a lot of data, I suppose you mean the Statistics section?

**Comment by Ysurac** [comment]
yes

**Comment by tonkepen100** [comment]
Well I start to study this, and that what I just said is putting a hundred, hahaha, how great you are.
Tomorrow I put exact relationship of feasible data for statistics.

Goodddddddd.

**Comment by tonkepen100** [comment]
But a clause, it would first essential to have, not callsing flight made by the pilot, if not the possibility of having either the identifier of its VA, or other pages that use identifiers of its flights. ESSENTIAL.

For example in my case www.piro.biz:

Pilots and controllers are determined by the initial indicative PIR, followed by his number ...

Lorem PIR0234 = Lorem Itsum Viva Spain.

So more or less all VAs, see the tables in their callsing essential would put it in such a code, since so

 the client knows that total flights have.

in fact you already have the data already on the popup of your flight.

Soory more for my english

**Comment by bilkaPek** [comment]
this is what you want ( tonkepen100 ) 
you want the same thing, with your VA link ??? 
![capture d ecran1](https://cloud.githubusercontent.com/assets/16308214/12043080/81493552-ae81-11e5-94c0-06e15626689b.png)
![capture d ecran2](https://cloud.githubusercontent.com/assets/16308214/12043081/814b36a4-ae81-11e5-9ffa-6c24c6653088.png)

**Comment by tonkepen100** [comment]
1 sec thanks and explain..

**Comment by tonkepen100** [comment]
No, forgive me if it already works perfectly. thanks, (which incidentally Photo left), easy fix.

this is the table ...

![captura](https://cloud.githubusercontent.com/assets/4571086/12043599/e5d251fc-ae86-11e5-8c47-947cfd74f48a.JPG)

another column where this your VA ID more or less so ....

![captura2](https://cloud.githubusercontent.com/assets/4571086/12043604/f0cff1e0-ae86-11e5-86bc-704b3bb4044f.JPG)


This is essential as a pilot, look inside your indicative of his will, and not by that callsing flown.

In fact in the popup on the map and goes below that number is only going have to create a new column and naturally searchable later.

ESSENTIAL

**Comment by tonkepen100** [comment]
It is that what got today my friend and Surac, left me so happy, and so hard to follow, my emotion is so brutal, I can not follow.
I am, and I know that on this issue Ysurac I get a hundred. I am very happy, dammit.
Pensare tomorrow, but today the two have made me happy. really

**Comment by Ysurac** [comment]
I will add pilot and id for IVAO & VATSIM, and owner instead for real flights.

**Comment by bilkaPek** [comment]
if you look closely the two above image, I put my 2 Information on "driver name and ID"
thou only wants ID

**Comment by Ysurac** [comment]
I will do the same, I like to see the name when available.

**Comment by bilkaPek** [comment]
ce qui serai bien c'est d'ajouter dans setting.php

$globalMyVaUrl = array('http://www.MyvirtualAirline.com/action.php/acars/data');
$globalVatSimUrl = array('http://www.vataware.com/pilot/');
$globalIvaoUrl = array('http://api.ivao.aero/getdata/whazzup/whazzup.txt');
afin de faciliter le choix pour le pilote

**Comment by tonkepen100** [comment]
$globalIvaoUrl = array('http://api.ivao.aero/getdata/whazzup/whazzup.txt');

jjajaajajajaja , caput ip

**Comment by tonkepen100** [comment]
10 minutes or caput ip

**Comment by bilkaPek** [comment]
I sui wrong in my line ;)

**Comment by tonkepen100** [comment]
not right, but I see you do not know the rules very well ivao, regarding wazzup,

I already took like 30 bans IPs, to reduce the 10 minutes of charging.

My solution is now 2 minutes, but that if high costs, delivering from 5 different ips, calling every 10 minutes and share the load between them for 2 minutes. juasssssss

**Comment by tonkepen100** [comment]
try it calls every two minutes ivao wazz, and tell me the result, I do not give more than two days

**Comment by bilkaPek** [comment]
2 weeks without any problems

**Comment by tonkepen100** [comment]
for then I surrender to you

**Comment by bilkaPek** [comment]
I do not think you'd be stupid
how you recovered the file whazzup
by cron task by our scripts deamon

**Comment by tonkepen100** [comment]
A, I do not forgive se.Yo'm not a programmer, I'm just a simple businessman with two companies and only 645 people in my charge. I have no idea of these things, only to win and give money to my staff.

The remaining piro only hobby.

**Comment by tonkepen100** [comment]
Jusqu'à ysurac m'a enseigné comme un cron de ces ensembles, souffert et souffert sans fin.
Mais la vérité est qu'il est un homme avec de la patience, pour moi, une grande personne, a subi ce que vous ne pouvez pas me l'imaginer. jours de Ysurac tous pouvaient le faire?, pourriez-vous, pourriez-vous, et tous eu absolument tout pour. une autre chose est alors mis me connaître.
Tout est un indice putain, mais il fonctionne toujours

**Comment by bilkaPek** [comment]
como eres español te respondo en tu idioma más facil para que me entiendas
No falte respeto a nadie... pues trata me igual y respónderme de esta manera no es agradable!
yo te agradece que me mostró mi error (pero asi nos quedamos...)
Yo he hecho solo una pregunta ( yo no te trate de idiota )
aqui se viene a "compartir" que sea programador o no y punto.
si eres frustrado es tu problema y no permitire a nadie que me falte respeto... 

&& Ysuarac desole d''avoir pourris ton hub &&

**Comment by tonkepen100** [comment]
Aucun homme, je lui ai dit que je suis un idiot, pas vous, et mon intérêt était jamais vous traite mal, ne le prenez pas pour que je puisse être l'homme le traducteur google.
--------------------------------------------------------------------------------------
No hombre no, lo que dije es que yo era idiota, no tu, y nunca fue mi interes tratarte mal, no te lo tomes asi que puede ser el traductor de google hombre.

**Comment by bilkaPek** [comment]
Igual tiene razón ... pues lo dejamos así mal entendido por culpa de Google

**Comment by tonkepen100** [comment]
-- Me and the main theme of the statistics, would be if the pilot has reached or not, I mean I explained, in real life and thank god aircraft reach their destination, but there are many virtual flights canceled for various reasons, although this issue for me is important because all flights at this time are recorded as realized, what I would do is, as I have in pyro is saying that it not not show, or holy column that says = flight made or flight unrealized.

Again attention I use google and you know as a theme, and again I repeat that under no circumstances was my intention, backwards it is a pleasure to see that you are interested, since more or less, I was alone here, with my few computer skills, did you feel hurt if I'm really sorry.

**Comment by tonkepen100** [comment]
Continuing with the thread of ideas for more statistics here are:

As ysurac think will put the issue of the calssings goes, then if you could put two new fields,,, please look photo

![captura](https://cloud.githubusercontent.com/assets/4571086/12049287/861bd436-aee5-11e5-8869-f15789ade191.JPG)


1.- PILOTS
2.- TOTAL FLIGHTS 

for example

**Comment by bilkaPek** [comment]
English :Transalte by Google
the idea that you had a profile page is to represent the pilot with information ??? or in the general stats page

Francais
l'idée que tu a est pour une page profile représentent le pilote avec ses informations  ??? ou dans la page générale de stats

la idea de que tu tines es una página de perfil por el piloto con las información del mismo ??? o en la página general de estadísticas

**Comment by tonkepen100** [comment]
If every driver has his own statistics page, then it would be a marvel, because then would take away pyro, leaving only the radar, but of course that would have to have a proper record of each pilot, for example to ensure data privacy and all that stuff.

I would start by the statistics section and expand the data in the columns of the tables that correspond to them.

Statistics give account that level, it is possible endless possible data, schedules, landing rate, number of flight hours, total time, split times, connected hours, played airports, finalized flights, incomplete flights, amount of fuel, that avion, hours VFR, IFR hours, maximum altitude, minimum altitude, accounting, etc etc, and an impressive number of possible data.

If that little by little is introduced, then you will have the best product at the virtual level. I'm telling you that.

I think we first have to define exactly what level statistics data can be entered, and once made a list of possibilities, then I think it is your turn, it can be implemented, since you are the kings of computer.

**Comment by Ysurac** [comment]
How can I know a flight is finalized with IVAO & VATSIM ? I have no data like "on ground" (available for real flight). The only think I saw for now is to check if final latitude/longitude are near an airport.

**Comment by tonkepen100** [comment]
OK, so vatsim as shown ivao state data generated by a letter, you step picture of the possible letters state, if the basic is 3, but there are exactly 12, in ivao and vatsim, let me a few seconds and you I do the exact relationship in a list.

**Comment by bilkaPek** [comment]
Eng:
you would not it be easier to recover data in the DB from VA ????
with flight status and information concerning the pilot ???

Fra:
ne serait t'il pas plus facile de récupérer les données de DB de la VA ????
avec les statut de vol et les informations le concernant le pilote..

Esp:
¿No sería más fácil recuperar los datos en la base de datos de VA ????
con el estado del vuelo y las información del piloto ???

**Comment by tonkepen100** [comment]
Bilka: And I always thought that, in itself is very simple, and also believe that the consumer would currently cron creates a lot lighter.

**Comment by Ysurac** [comment]
How and where ? The only info I have come from whazzup.txt for IVAO or vatsim-data.txt for VATSIM.

**Comment by bilkaPek** [comment]
Eng:
that we look at the file and VATSIM ivao it gives no information knew flight status
I think you need a reason ysurac do with latitude / longitude are near an airport otherwise I do not see another solution yet

Fra
qu'en on regarde les fichier ivao et vatsim il ne donne aucune information sut le statut de vol
je pense que tu a raison ysurac  faut voir avec la latitude / longitude sont près d'un aéroport sinon je ne vois pas d'autre solution pour le moment....

**Comment by tonkepen100** [comment]
if it is given, but as the state pyro Please take us?

![captura](https://cloud.githubusercontent.com/assets/4571086/12054380/78a22d94-af23-11e5-9d0d-7be404b00346.JPG)

**Comment by tonkepen100** [comment]
but if you mean the flight status made, or not, that if that goes for program code and is carried by the coordinates of the arrival airport, in my case I put 5 miles, if the flight does not go into those five miles soIt is given as not

**Comment by Ysurac** [comment]
OK, I use coordinates, this seems to work. I test it a little more and I will commit.
This also work for real flights, so this is good (for real flights I also use altitude).

**Comment by tonkepen100** [comment]
Fantastic

**Comment by tonkepen100** [comment]
Ysurac all information borrare morning flight to begin by calendar year, that table was the one who had to empty? ... Spotter_output perhaps ?.
I know this is not going well here but do not open another thread, thanks.

**Comment by Ysurac** [comment]
You can run something like this : 
**DELETE FROM spotter_output WHERE date < '2016-01-01 00:00:00';**

**Comment by tonkepen100** [comment]
ahhh I understand this is making it clear previous flights by 2016, thus not have to empty the table or anything, thank you, thank you very much good idea.

**Comment by tonkepen100** [comment]
Returning to the thread (Recollect that I write with google):

![captura](https://cloud.githubusercontent.com/assets/4571086/12062982/cba9f39a-afa6-11e5-97bb-f6ca08e23673.JPG)

FLIGHTS CURRENTS IN SECTION

 I personally would leave the following changes ....

1. ORDER OF COLUMNS AND DATA SHOW

Photo-Airline-PILOT-Callsing-Aircraft-DEPARTURE-ARRIBAL-ETE-FPALTITUDE-STATUS

A. A new column is added or the name of the pilot or the number of VA.
B. The name of the columns Coming to-Flying to, is amended by departure and arribal.
C. The ETE-altitude Flight plan fields and status flags with either image or named are included.

I would think for that section the basic data and more used by pilots to said section was appropriate both to ivao vatsim.

Naturally here also would fail to include the possible drivers at that time were connected, not them aside, poor things are always the most forgotten. Hahaha.

**Comment by tonkepen100** [comment]
And continuing here you have some pictures, with possible areas of possible statistics. I put on if you see something interesting that could include attention absolutely all the IVAO and VATSIM data delivery, so all are possible and work. clarify that the Accounting Section of the pilot m not naturally vatsim ivao or delivery, it is a proper script.

![captura](https://cloud.githubusercontent.com/assets/4571086/12063437/e2b8746e-afae-11e5-9245-ddf38e0692b9.JPG)

![captura2](https://cloud.githubusercontent.com/assets/4571086/12063439/e9393cec-afae-11e5-99db-6962a240bd28.JPG)

![captura3](https://cloud.githubusercontent.com/assets/4571086/12063441/edb83872-afae-11e5-8547-12567805bc93.JPG)

![captura4](https://cloud.githubusercontent.com/assets/4571086/12063443/f3de7c02-afae-11e5-8e9e-87d95522ad6f.JPG)

![captura5](https://cloud.githubusercontent.com/assets/4571086/12063445/f81f747e-afae-11e5-9cb2-59c1ca9cbac3.JPG)

![captura6](https://cloud.githubusercontent.com/assets/4571086/12063446/fbfc4c5c-afae-11e5-94ed-6820a9a30e2f.JPG)

![captura7](https://cloud.githubusercontent.com/assets/4571086/12063448/ffb05c4e-afae-11e5-9e7d-374bf10398c9.JPG)

![captura8](https://cloud.githubusercontent.com/assets/4571086/12063450/0319193e-afaf-11e5-9710-f8b072e213dc.JPG)

As you can see the possibilities are limitless

Regards

**Comment by Ysurac** [comment]
I will try to put some of them. I like the altitude & speed graph for example (I have the altitude graph, but it's a good idea to have both on same graph).

**Comment by tonkepen100** [comment]
I love this answer Ysurac, thanks, if you need some code or something let me know and I'll be happy to send it,

**Comment by tonkepen100** [comment]
I also think it could be interesting in the statistics section, as there is a flight chart the last 7 days putting out another po every month. would be fantastic.

![captura](https://cloud.githubusercontent.com/assets/4571086/12071216/0028686e-b0a0-11e5-90ef-f45f93886eb8.JPG)

**Comment by tonkepen100** [comment]
yurac exceptional work that has given a dramatic change, my most sincere congratulations for your work. but I think in the remaining gap could fit in the month that is the only thing missing, so would be complete,

![captura](https://cloud.githubusercontent.com/assets/4571086/12201488/041cc2e6-b625-11e5-9ecc-02f988cf0e17.PNG)
